src/pilot/templates/nic-configs/5_port/controller.yaml

```diff
@@ -168,11 +201,13 @@ resources:
             -
               type: ovs_bridge
               name: br-tenant
+              mtu: {get_param: DefaultBondMTU}
               members:
                 -
                   type: linux_bond
                   name: bond0
                   bonding_options: {get_param: ControllerBondInterfaceOptions}
+                  mtu: {get_param: DefaultBondMTU}
                   members:
                     -
                       type: interface
```

Comment:
I think we missed setting mtu on the two controller interfaces below.

Reply:
will add the mtu settings for interfaces as well